clientd3d/drawbsp.c

```diff
@@ -23,7 +23,7 @@
 
 #include "client.h"
 
-#define MAX_OBJS_PER_LEAF 100
```

Comment:
Increasing the max object limit is perhaps OK, but the O(N^2) bubble sort on line 2271 needs to be replaced if so.  Let's put in std::sort instead.